_sass/partials/_typography.scss

```diff
@@ -309,8 +310,8 @@ caption {
 }
 
 blockquote {
-  border-left: 2px solid #bbb;
-  color: #888;
+  border-left: 2px solid $brand-red;
+  color: $brand-gray80;
```

Comment:
brand-gray80 is accessible, when font-size is 16.5, now it's 16, so nearly there:
<img width="709" alt="Screenshot 2023-06-28 at 13 27 12" src="https://github.com/solita/solita.github.com/assets/52316685/c8b3e94f-a0a6-4269-91e0-9bd16fc6b9b1">